docker-compose.minimal.yaml

```diff
@@ -71,11 +73,15 @@ services:
     - --public-key=${MINER_PUBLIC_KEY}
     - --node=bootstrap-node:1848
     - --worker=constant-delay
+    - --constant-delay-block-time=15
```

Comment:
why do we want to increase this?

Reply:
I guess with fast-development, it doesn't matter much. But I'd still like to know the rationale.

Reply:
Just a git issue, will revert.